modules/bloxone_infra_host_aws/versions.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_version = ">= 1.5"
```

Comment:
Any reason why we are restricting to these versions?

Reply:
removed versions.